pyproject.toml

```diff
@@ -345,6 +351,9 @@ lint.ignore = [
     "D415",  # First line should end with a period, question mark, or exclamation point
     ]
 
+[tool.ruff.lint.per-file-ignores]
+"tests/unit*" = ["S604", "S605", "S318", "S108"]  # Less strict security checks in tests/
```

Comment:
@martinhoyer can we list this out nicely with comments?

Reply:
Addressed in https://github.com/teemtee/tmt/pull/2842/commits/706c670a649614f3d616e80b39120af29202101f